KsApi/Service.swift

```diff
@@ -97,13 +97,19 @@ public struct Service: ServiceType {
   }
 
   public func changeEmail(input: ChangeEmailInput) ->
-    SignalProducer<GraphMutationEmptyResponseEnvelope, GraphError> {
-    return applyMutation(mutation: UpdateUserAccountMutation(input: input))
+    SignalProducer<UpdateAccountEnvelope, ErrorEnvelope> {
```

Comment:
Same idea I mentioned in the MockService makes sense here. Looks like each of these methods is doing the exact same thing so they can probably be consolidated into one method? What do you think?

Reply:
See novel above :)